xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/macro/include/IncludeMacroParameters.java

```diff
@@ -72,6 +74,8 @@
      * @since 3.4M1
      */
     @PropertyDescription("the reference of the resource to display")
+    @PropertyGroup({"stringReference"})
```

Comment:
You don't need the {} when there is only one.

Reply:
Good to know, I thought this feature had to be implemented in the annotation itself.